core/src/main/java/nl/nn/adapterframework/configuration/Configuration.java

```diff
@@ -433,6 +433,13 @@ public void removeStopAdapterThread(Runnable runnable) {
 		adapterManager.removeStopAdapterThread(runnable);
 	}
 
+	/**
+	 * Include the referenced Module in this configuration
+	 */
+	public void registerInclude(Include module) {
+		// method exists to trigger FrankDoc.
+	}
+
```

Comment:
Ik zou het enorm op prijs stellen als hier nog een oplossing voor bedacht wordt. Dit zou uit de digester-rules.xml gehaald moeten (kunnen) worden. Zonder dat er lege setters nodig zijn in de Configuration klasse.

Reply:
Deze setter geeft aan op welke positie in de xml een Include mag voorkomen, en geeft de beschrijving in de FrankDoc